practice-one/src/index.html

```diff
@@ -4,10 +4,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/global.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/header.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/footer.css" type="text/css"/>
-    <link rel="stylesheet" href="assets/css/reset.css" type="text/css"/>
     <link rel="stylesheet" href="assets/css/text.css" type="text/css"/>
```

Comment:
@HoangLiarPhuongLe  Is it appropriate to import text styles after the header and footer?

Reply:
@HoangLiarPhuongLe In this file, are you only styling text? If not, why is it named `text.css`?